docs/src/userguide/language_basics.rst

```diff
@@ -326,12 +316,29 @@ and is typically what one wants).
 If you want to use these numeric Python types simply omit the
 type declaration and let them be objects.
 
+Extension Types
+---------------
+
 It is also possible to declare :ref:`extension-types` (declared with ``cdef class`` or the ``@cclass`` decorator).
-This does allow subclasses. This typing is mostly used to access
-``cdef``/``@cfunc`` methods and attributes of the extension type.
+Those will have a behaviour very close to python classes (e.g. creating subclasses), but are faster. Typing variable
```

Comment:
```suggestion
Those will have a behaviour very close to python classes (e.g. creating subclasses), but are faster. Typing a variable
```

Reply:
```suggestion
Those will have a behaviour very close to python classes (e.g. creating subclasses), but access to their members is faster from Cython code. Typing variable
```

I'm not quite sure what it means for a class to be "faster" so I made it a little more specific